### PR TITLE
Add NPC section

### PR DESCRIPTION
**Pull request by agude**
This is going to need iteration! I can't get away from the kids long enough to really get into the writing flow, but better to have some words down than none!

### CODE REVIEW

File: _chapters/03-character-creation.md [hunk 1]
```diff
@@ -131,10 +131,35 @@ take in response to the story.
 
 ## Non-Player Characters
 
-<!-- TODO: Write this. -->
-
-<!-- TODO: Do groups go here? -->
-
+NPCs are the characters in the story that fill out the world. They're played
```
Comment (charles-uno) [inline _chapters/03-character-creation.md:134]: I haven't even read a word yet and this is way too long. (Which is fine... better too long than too short pre-iteration)

Comment (agude) [reply _chapters/03-character-creation.md:134]: Yeah it is long. Is it better to split example into a:

> Example!
>
> More

Section?

Comment (charles-uno) [reply _chapters/03-character-creation.md:134]: I think a single NPC section is appropriate, and I do think it should include a few examples. 

This has already been in limbo for a few days, and I don't have the brainpower to iterate on it at the moment. My inclination would be get it merged and iterate more later

**Review by charles-uno — APPROVED** [review]

File: _chapters/03-character-creation.md [hunk 2]
```diff
@@ -131,10 +131,35 @@ take in response to the story.
...
+challenge the group of PCs should have a larger skill pyramid than PCs, as
+they're competing against the entire group.
+
+Background characters show up once or twice an general serve a simple role in
+the narrative. They can generally get away with a single aspect and a small
+selection of skills. The bartender who makes small talk and drops a few rumors
+doesn't need a full skill Pyramid, he probably just needs `Performance` or
+`Persuasion`. Likewise, a set of guards keeping Tor Byron out of the royal
+ball might only need an aspect like ~~Somewhat Loyal and Underpaid Guards~~
+and the skills `Fight` and `Persuasion` in case Tor Byorn tries to force or
```
[the remark this answers is not in the document]
Comment (agude) [reply _chapters/03-character-creation.md:151]: 😵‍💫 maybe?

You defend with right, but you probably dodge shoot with agility?

**Pull request merged**